Modules/Quotas/src/Quotas.Application/Identities/Queries/GetQuotasForIdentity/ListQuotasForIdentityResponse.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Backbone.BuildingBlocks.Application.Pagination;
+using Backbone.Modules.Quotas.Application.DTOs;
+
+namespace Backbone.Modules.Quotas.Application.Identities.Queries.GetQuotasForIdentity;
+
+public class ListQuotasForIdentityResponse
```

Comment:
Given we have this response, the resulting HTTP response content looks as follows:

```json
{
    "result": {
        "items": [...]
    }
}
```

But what we want is:

```json
{
    "result": [...]
}
```

Have a look at other routes that return multiple items to see how it's done.